cabal.project.freeze

```diff
@@ -212,7 +212,7 @@ constraints: any.Boolean ==0.2.4,
              any.old-time ==1.1.0.3,
              any.optparse-applicative ==0.17.0.0,
              optparse-applicative +process,
-             any.pact ==4.6.0,
+             any.pact ==4.7.0,

```

Comment:
@larskuhtz are we good to modify the freeze file?

Reply:
I'll ask Louis.

Reply:
Yes we are, we haven't checked in the release freeze file yet.